src/Microsoft.Extensions.DependencyModel/Library.cs

```diff
@@ -10,6 +10,11 @@ namespace Microsoft.Extensions.DependencyModel
     public class Library
     {
         public Library(string type, string name, string version, string hash, IEnumerable<Dependency> dependencies, bool serviceable)
```

Comment:
NIT: break the lines into 120 columns please. If you do that, put each parameter in its own line. Like:

public Library(string type,
                        string name,
                        ...)